app/assets/javascripts/components/select-with-search.js

```diff
@@ -20,7 +20,14 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.choices = new window.Choices(this.select, {
       allowHTML: true,
       searchPlaceholderValue: 'Search in list',
-      shouldSort: false // show options and groups in the order they were given
+      shouldSort: false, // show options and groups in the order they were given
+      itemSelectText: '',
+      searchResultLimit: 100,
```

Comment:
This is a completely arbitrary number. It could just as easily be 20 or 50. Happy to go with whatever the consensus is.

Reply:
100 seems a perfectly reasonable choice to me! 👍🏻 

Ideally we wouldn't go for _much more_ than 100, though, because we know Choices.js can get a little janky when rendering 100's of results at once. For example, I expect it'll feel a bit jankier if rendering 500 items.